lib/modules/@apostrophecms/doc-type/src/apos/components/AposDocEditor.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <AposModal
+    class="apos-doc-editor" :modal="modal"
+    :modal-title="modalTitle"
+    @inactive="modal.active = false" @show-modal="modal.showModal = true"
+    @esc="cancel" @no-modal="$emit('safe-close')"
+  >
+    <template #primaryControls>
+      <AposButton
+        type="primary" label="Save"
+        @save="submit"
+      />
+    </template>
+    <template #leftRail>
+      <AposModalRail>
+        <AposModalTabs
+          :current="currentTab" :tabs="tabs"
+          @select-tab="switchPane"
+        />
+      </AposModalRail>
+    </template>
+    <template #main>
+      <AposModalBody>
+        <template #bodyMain>
+          <AposModalTabsBody>
+            <div class="apos-doc-editor__body">
+              <AposSchema
+                :schema="currentFields" :doc="myDoc"
+                @input="update"
+              />
+            </div>
+          </AposModalTabsBody>
+        </template>
+      </AposModalBody>
+    </template>
+    <template #rightRail>
+      <AposModalRail type="right">
+        <div class="apos-doc-editor__utility">
+          <AposSchema
+            :schema="utility" :doc="myDoc"
+            @input="update" :modifiers="['small', 'inverted']"
+          />
+        </div>
+      </AposModalRail>
+    </template>
+  </AposModal>
+</template>
+
+<script>
+import AposModalParentMixin from '../../../../modals/src/apos/mixins/AposModalParentMixin';
```

Comment:
The `'Modules/@apostrophecms'` alias wasn't working for me for script imports.

Reply:
We gotta fix this. I'll get in.